build.xml

```diff
@@ -271,6 +271,14 @@
             <arg value="index.js"/>
         </exec>
     </target>
+    
+    <target name="generate-images-py">
+        <exec executable="python3" logerror="true" dir="${basedir}/utils/examples" failifexecutionfails="true" failonerror="true">
```

Comment:
In some cases `python` is still aliased to Python 2. `python3` is generally favoured by most package managers as the preferred name for Python 3 installations. (That said, if you have a virtual environment you will have access to both the `python` and `python3` executables, since `python` is an alias to `python3`.)

Reply:
I see your point, thanks.